hazelcast-jet-core/src/main/java/com/hazelcast/jet/core/Processor.java

```diff
@@ -99,24 +99,34 @@
 
     /**
      * Tells whether this processor is able to participate in cooperative
-     * multithreading. This means that each invocation of a <em>processing
-     * method</em> in this class will take a reasonably small amount of time
-     * (up to a millisecond). Violations will manifest themselves as increased
-     * latency due to slower switching of processors.
-     * <p>
-     * A cooperative processor should also not attempt any blocking operations,
-     * such as I/O operations, waiting for locks/semaphores or sleep
-     * operations. Violations of this rule will manifest as less than 100% CPU
-     * usage under maximum load (note that this is possible for other reasons too,
-     * for example if the network is the bottleneck or if {@linkplain
-     * JetProperties#JET_IDLE_COOPERATIVE_MAX_MICROSECONDS parking time} is too high).
-     * The processor must also return as soon as the outbox rejects an item
-     * (that is when the {@link Outbox#offer(Object) offer()} method returns
-     * {@code false}).
-     * <p>
-     * If this processor declares itself cooperative, it will share a thread
-     * with other cooperative processors. Otherwise it will run in a dedicated
-     * Java thread.
+     * multithreading. If this processor declares itself cooperative, it will
+     * share a thread with other cooperative processors. Otherwise it will run
+     * in a dedicated Java thread.
+     * <p>
+     * There are specific requirements that all <em>processing methods</em> of
+     * a cooperative processor must follow:<ul>
```

Comment:
can you fix the formatting here? looks weird? See for example on `Jet.bootstrappedInstance`